src/tools/illink/src/ILLink.RoslynAnalyzer/DataFlow/LocalDataFlowVisitor.cs

```diff
@@ -190,6 +195,11 @@ TValue ProcessSingleTargetAssignment (IOperation targetOperation, ISimpleAssignm
 						// For now, just don't warn. https://github.com/dotnet/linker/issues/2731
```

Comment:
```suggestion
						// For now, just don't warn. https://github.com/dotnet/runtime/issues/93277
```

I moved the issue to runtime - and we should look into fixing this.